### PR TITLE
added API topic

### PR DESCRIPTION
**Pull request by drothery-edb**
Added API topic

- [x ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+```
+curl https://portal.edbcloud.com/api/v1/auth/provider
+```
+    
+The response returns the `clientId`, `issuerUri`, `scope`, and `audience`. For example:
+```
+{
+  "clientId": "pM8PRguGtW9yVnrsvrvpaPyyeS9fVvFh",
+  "issuerUri": "https://edbcloud.us.auth0.com",
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:29]: plz change this to: `https://auth.edbcloud.com`, we are going to use customized domain name

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  "audience": "https://portal.edbcloud.com/api"
+}
+```
+
+## Request the Device code
+
+Use the following call to get a device code: 
+```
+   curl --request POST \
+     --url 'https://edbcloud.us.auth0.com/oauth/device/code' \
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:40]: plz change this to: `https://auth.edbcloud.com/oauth/device/code`, we are going to use customized domain name

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+- `verification_uri_complete` - the complete URL you use to authorize the device. You can use this URL to embed the user code in your app's URL.
+- `expires_in` indicates the lifetime (in seconds) of the `device_code` and `user_code`.
+- `interval` indicates the interval (in seconds) at which the app should poll the token URL to request a token.
+
+For example: 
+```
+{
+  "device_code": "KEOY2_5YjuVsRuIrrR-aq5gs",
+  "user_code": "HHHJ-MMSZ",
+  "verification_uri": "https://edbcloud.us.auth0.com/activate",
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:61]: plz change this to: `https://auth.edbcloud.com/activate`, we are going to use customized domain name

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  "expires_in": 900,
+  "interval": 5,
+  "verification_uri_complete": "https://edbcloud.us.auth0.com/activate?user_code=HHHJ-MMSZ"
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:64]: plz change this to: `https://auth.edbcloud.com/activate?user_code=HHHJ-MMSZ`, we are going to use customized domain name

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+6. Log in with your Azure AD credentials.
+ 
+
+## Request the Token
+
+Use the `curl --request POST` command to request a token. For example, 
+```
+curl --request POST \
+  --url 'https://edbcloud.us.auth0.com/oauth/token' \
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:83]: plz change this to: `https://auth.edbcloud.com/oauth/token`, we are going to use customized domain name

File: product_docs/docs/edbcloud/beta/reference/index.mdx [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+## Refreshing your Token
+
+You can use the refresh token to get a new access token. Usually you need a new access token only after the previous one expires or when gaining access to a new resource for the first time. You shouldn't call the endpoint to get a new access token every time you call an API. There are rate limits that throttle the amount of requests to the endpoint that can be executed using the same token from the same IP. 
+
+To refresh your token, make a POST request to the `/oauth/token` endpoint in the Authentication API, using `grant_type=refresh_token`. For example:
+
+```
+curl --request POST \
+  --url 'https://edbcloud.us.auth0.com/oauth/token' \
```
Comment (Anhui-tqhuang) [inline product_docs/docs/edbcloud/beta/reference/index.mdx:159]: plz change this to: `https://auth.edbcloud.com/oauth/token`, we are going to use customized domain name

Comment (drothery-edb) [reply product_docs/docs/edbcloud/beta/reference/index.mdx]: See latest commit

**Review by xugy99 — APPROVED** [review]
NICE!

**Review by ba-edb — APPROVED** [review]
LGTM, thanks!

**Pull request merged**